ballerina/listener.bal

```diff
@@ -86,6 +86,12 @@ public class Listener {
         __Schema & readonly schema = engine.getSchema();
         __Type? subscriptionType = schema.subscriptionType;
         if subscriptionType is __Type && self.wsListener is () {
+            string httpVersion = self.httpListener.getConfig().httpVersion;
+            if httpVersion != http:HTTP_1_1 {
```

Comment:
Shouldn't we allow `HTTP_1` as well?

Reply:
Checked `HTTP_1` listener and it's worked as expected. Therefore allowed `HTTP_1` as well. fixed in https://github.com/ballerina-platform/module-ballerina-graphql/pull/1034/commits/dd7cd31445af6c24613ebece51e5f319ff595fa4